### DISCUSSION TITLE
Aufgabenverteilung

### DISCUSSION THREAD

**Discussion by Matt-17** [Wendelpunkt/ToypadChallenge#3] [Ideas]
Jedes Feld auf dem Pad steht für eine Aufgabe wie zum Beispiel Müll rausbringen, Pflanzen gießen. Dazu hat man Figuren die je einer Person im Team/Familie/WG entsprechen. Stellt man die Figur auf eine Aufgabe, wird die Person damit verknüpft und das Feld leuchtet rot. Idealerweise wird dies auf einem Monitor live dargestellt. Ähnlich wie auf einen Aufgabenboard kann man dann Aufgaben als in Bearbeitung oder fertig definieren, womit das Feld auf gelb bzw. Grün springt (falls das so detailliert möglich ist).

**Comment by tomwendel** [comment]
Ich glaub das hab ich noch nicht ganz gecheckt. Aber der spielerische Absatz bei Haushaltsaufgaben klingt schon mal cool.

**Comment by pherting** [comment]
Ich hatte eine ähnliche Idee 😂
Man könnte die Aufgaben auch auslösen, man stellt drei Figuren auf, es beginnt eine roling animation und irgendwann bleibt es bei einer Farbe stehen bzw. Einer Platform und der jenige ist dran. Müsste man überlegen wie man es mit mehr als drei Spielern machen könnte.. aber mals in den Raum geworfen 😅

**Reply by Matt-17** [reply]
Ganz gute Idee, aber ich hatte schon die letzten 24h in eine etwas andere Richtung überlegt. Ich glaube das könnte aber eine gute Alternatividee im gesamten werden.

**Comment by Matt-17** [comment]
Nun habe ich meine Idee mal etwas konkretisiert und den (mir neuen) Rahmenbedingungen des Toypads angepasst. 

Also, die Idee ist weiterhin eine Aufgabenverteilung, aber etwas spielerisch (also mit Gamification).

Auf die linke Seite des Pads werden nun also 1-4 (oder falls Tom noch einen Reader findet, weitere) "Aufgaben" gelegt. Das können im Haushalt meinetwegen 

- Müll rausbringen
- Wäsche waschen

oder auf Arbeit halt

- Tests schreiben
- Implementierung starten
- Kaffee trinken

Für jede dieser Aufgaben braucht man entsprechende Tags, die dank LEGO ja sehr kreativ sein dürfen. Weiterhin braucht man für jede Person einen Tag. 

Nun kommt jemand und sagt, ich will jetzt den Müll rausbringen. Also stellt diese Person seinen Tag auf das Center. Nun ist man im Bearbeiten-Modus. Nun bewegt man die gewünschte Aufgabe nach rechts. Sie ist nun in Bearbeitung und man nimmt sein personalisiertes Tag wieder weg. 

Hat man die Aufgabe später erledigt, stellt man wieder seine Person drauf und entfernt dann die Aufgabe gänzlich vom Toypad. 

Im Programm gibt es dafür nun für die Person einen Punkt. 

Ab jetzt folgen zusätzliche Features und Idee
- nicht angefangene Aufgaben zum Zeitpunkt x bedeuten für jeden 2 Minuspunkt
- Aufgaben in Bearbeitung zum Zeitpunkt x bedeuten für die Person 1 Minuspunkt
- individuelle Farbei beim Auflegen der Person
- schöne Leuchtanimation wenn eine Aufgabe den Status wechselt
- roten Blitzen, wenn die falsche Person eine Aufgabe aus der Bearbeiten entfernt
- Administratorperson braucht es evtl.
- ...

So, ich hoffe es ist etwas konkreter und verständlicher geworden :)
